includes/classes/class-omise-card-image.php

```diff
@@ -12,7 +12,7 @@ class Omise_Card_Image {
 		 */
 		private static function get_image( $file, $alternate_text ) {
 			$url = WC_HTTPS::force_https_url( WC()->plugin_url() . '/assets/images/icons/credit-cards/' );
-			return "<img src='$url/$file' width='38px' alt='$alternate_text' />";
+			return "<img src='$url/$file' style='width: 38px;' alt='$alternate_text' />";
```

Comment:
@guzzilar I agree with the solution to provide a style class for the image. It deliver the benefit to customer. But the redundant configuration is unnecessary.

If the customer really want to customize, they can customize it by using that solution or by using so many other flexible CSS implementation style.